app/experimenter/kinto/tasks.py

```diff
@@ -35,26 +35,26 @@ def nimbus_push_experiment_to_kinto(experiment_id):
 
     try:
         experiment = NimbusExperiment.objects.get(id=experiment_id)
-        logger.info(f"Pushing {experiment} to Kinto")
+        logger.info(f"Pushing {experiment.slug} to Kinto")
```

Comment:
Oh yeah I just realized since remote settings uses the slug as its identifier it's easier to look at the logs with slugs and compare them to what the api is saying instead of the name.